src/components/Footer/FooterSocial/FooterSocialItem.tsx

```diff
@@ -1,15 +1,26 @@
 import React from 'react';
 
+import Icon from 'Icon/Icon';
+
 type FooterSocialItemProps = {
   text: string;
   url: string;
 };
 
 export const FooterSocialItem = ({ text, url }: FooterSocialItemProps) => {
   return (
-    <a href={url} rel="noopener noreferrer" target="_blank">
+    <a
+      href={url}
+      className="footer-social-item"
+      rel="noopener noreferrer"
+      target="_blank"
+    >
+      <Icon
+        className="footer-social-item__icon"
+        name={text.charAt(0).toLowerCase() + text.slice(1)}
```

Comment:
could simplify to `text.toLowerCase()`

Reply:
I just need to make first character lowercase. There is an item with the uppercase in the middle of string which should not be transformed.

Reply:
👍